src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (...params) => {
+    if (params.length > 0) {
+      sum += params[0];
+
+      return adder;
+    } else {
+      const currentSum = sum;
+
+      sum = 0;
+
+      return currentSum;
+    }
```

Comment:
Else is redundant here
```suggestion
    }
    
    const currentSum = sum;

     sum = 0;

    return currentSum;
```